app/models/service_terraform_template.rb

```diff
@@ -39,9 +41,25 @@ def execute(action)
 
     task_id = MiqTask.generic_action_with_callback(task_opts, queue_opts)
     task    = MiqTask.wait_for_taskid(task_id)
+
+    $embedded_terraform_log.debug("Service(#{id}).execute(#{action}) ends, with task/#{task_id}:#{task}")
     raise task.message unless task.status_ok?
   end
 
+  def postprocess(action)
+    $embedded_terraform_log.debug("Service(#{id}).postprocess(#{action}) starts")
+    case action
+    when ResourceAction::RECONFIGURE
```

Comment:
Why case/when for a single branch versus just `if action == ResourceAction::RECONFIGURE` ?